Frends.AzureBlobStorage.DownloadBlob/Frends.AzureBlobStorage.DownloadBlob/Definitions/Source.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AzureBlobStorage.DownloadBlob
+{
+    /// <summary>
+    /// Input-class for DownloadBlob-task.
+    /// </summary>
+    public class Source
+    {
+        /// <summary>
+        /// Connection string to Azure storage.
+        /// </summary>
+        [DefaultValue("UseDevelopmentStorage=true")]
+        [DisplayFormat(DataFormatString = "Text")]
+        public string ConnectionString { get; set; }

```

Comment:
Sorry, forgot last time - this connection string should definitely be secret, as it contains access key.

Reply:
Fixed.